consensus/bundle_validator/bundle_validator.c

```diff
@@ -135,6 +135,7 @@ retcode_t iota_consensus_bundle_validator_validate(
   int64_t bundle_value = 0;
   flex_trit_t bundle_hash[FLEX_TRIT_SIZE_243];
   bool valid_sig = true;
+  Kerl kerl = {};
```

Comment:
call `init_kerl` on this?

Reply:
kerl has initialized in the bundle method.